svg/elements/feComposite.json

```diff
@@ -52,40 +52,52 @@
           "__compat": {
             "support": {
               "chrome": {
-                "version_added": "1"
+                "version_added": "1",
+                "notes": "BackgroundImage/BackgroundAlpha are not supported (see <a href='https://crbug.com/137230'>Chromium bug 137230</a>)."
               },
               "chrome_android": {
-                "version_added": true
+                "version_added": true,
+                "notes": "BackgroundImage/BackgroundAlpha are not supported (see <a href='https://crbug.com/137230'>Chromium bug 137230</a>)."
               },
               "edge": {
-                "version_added": "≤18"
+                "version_added": "≤18",
+                "notes": "BackgroundImage/BackgroundAlpha are not supported from version 79(see <a href='https://crbug.com/137230'>Chromium bug 137230</a>)."
               },
               "firefox": {
-                "version_added": true
+                "version_added": true,
+                "notes": "BackgroundImage/BackgroundAlpha are not supported (see <a href='https://bugzil.la/437554'>bug 437554</a>)."
               },
               "firefox_android": {
-                "version_added": true
+                "version_added": true,
+                "notes": "BackgroundImage/BackgroundAlpha are not supported (see <a href='https://bugzil.la/437554'>bug 437554</a>)."
               },
               "ie": {
-                "version_added": null
+                "version_added": true,
```

Comment:
I tested and this works (except for operator value `lighter`).  I have been told by Robert only edge prior to Blinks supports BackgroundImage/Alpha but have no issue to track - hence see note below has no bug number.